backend/tests/conferences/factories.py

```diff
@@ -66,6 +66,34 @@ def languages(self, create, extracted, **kwargs):
             for language_code in extracted:
                 self.languages.add(Language.objects.get(code=language_code))
 
+    @factory.post_generation
+    def submission_types(self, create, extracted, **kwargs):
```

Comment:
This fixture feels a lot like magic, maybe we should add some documentation around it to make it clear what it is doing :)

Reply:
Have a look now :)